.rubocop.yml

```diff
@@ -11,6 +11,7 @@ Layout/ParameterAlignment:
 
 Metrics/BlockLength:
   Exclude:
+    - ffi-gdal.gemspec
```

Comment:
In general I prefer to avoid adding single files to the rubocop config and do the disabling in the file itself.  It makes the config less cluttered, and you don't have to remember to clear it out if the file gets deleted.

Reply:
I get that. ...but in this case, the file is never gonna be deleted, and the cop is never going to be fixed (frankly, it's kinda silly that this isn't a default exclusion)...

Reply:
Good news!  It is a default exclusion!  https://github.com/rubocop/rubocop/blob/master/config/default.yml#L2505

You're just missing:
```
inherit_mode:
  merge:
    - Exclude
```